co-log/tutorials/1-intro/Intro.md

```diff
@@ -22,16 +20,13 @@ import Colog.Core (LogAction (..), (<&), logStringStdout)
 
 `co-log` is based on the following data type:
 
-```idris
```

Comment:
This was marked as `idris` intentionally. We're using `markdown-unlit` preprocessor that combines all code blocks with `haskell` tag. So the only way to ignore a code block is to specify it with a different language. And Idris has similar syntax, so we still can have syntax highlighting in this file.

Reply:
Got it. I somehow expected that you didn't unintentionally mix up Idris and Haskell :-)